### PR TITLE
PLSS Parcel Fabric Update Blogpost

### PR DESCRIPTION
**Pull request by jpechmann**
Pull request description
------------------------

### CODE REVIEW

**Comment by jpechmann** [comment]
@steveoh so ready!

File: _posts/2016-12-15-utah-plss-parcel-fabric-update.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- Featured
+tags:
+- public lands survey
+- SITLA
+- AGRC
+- cadastre
+
+---
+
+The final extract for the [Utah PLSS Parcel Fabric](http://mapserv.utah.gov/PLSS) has been delivered to AGRC by the BLM and its contractors. The PLSS map layer is now actively being maintained by AGRC, and work continues to convert existing PLSS survey control point documentation (tie-sheets) to digital formats in preparation for adjusting the Fabric in each county. We estimate the 40% of the existing control now has validated, digital geographic coordinates.
```
Comment (steveoh) [inline _posts/2016-12-15-utah-plss-parcel-fabric-update.md]: > We estimate the 40% of the existing control now has validated, digital geographic coordinates. 

This sounds funny

File: _posts/2016-12-15-utah-plss-parcel-fabric-update.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+AGRC is working with SITLA to add digital boundaries for the public and private land ownership and designations into the fabric one county at a time. Working in the parcel fabric allows any and all adjustments or additions to be applied at once to all the layers involved in the PLSS: Townships, Sections, Secondary Division, Special Surveys, and the Land Ownership features that SITLA contributes. Land ownership boundaries for Davis and Morgan counties have been added to the fabric and adjusted with the addition of the high precision survey control points. Summit County is currently in progress, with Sevier, Sanpete, and Cache counties on deck. The digital survey control for these counties has been compiled and validated and is ready for integration into the Fabric when they are ready.
+
+The Utah PLSS Parcel Fabric production version is available for download in several forms from the **[PLSS / GCDB Cadastre Data Page]({{ "/data/sgid-cadastre/plss/" | prepend: site.baseurl }})**. The file sizes are quite large, so please consider this when downloading.
+
+***Available for download:***
+
+- Utah PLSS Fabric 2.5 (1.75 GB zip file) the entire statewide fabric with an mxd
```
Comment (steveoh) [inline _posts/2016-12-15-utah-plss-parcel-fabric-update.md]: > Utah PLSS Fabric 2.5 (1.7...
should this be v2.5

File: _posts/2016-12-15-utah-plss-parcel-fabric-update.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+- Published CadNSDI V 2.5 (250 MB zip file) PLSS fabric data in the individual feature classes published in the CadNSDI (Townships, First Division, Second Division, Intersected, Special Survey, Meandered Water, Metadata at a Glance, Point.
```
Comment (steveoh) [inline _posts/2016-12-15-utah-plss-parcel-fabric-update.md]: k think v2.5 is more accepted when talking about versions

File: _posts/2016-12-15-utah-plss-parcel-fabric-update.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+- Published Utah PLSS V 2.5 (250 MB zip file) PLSS fabric data in the individual feature classes published in the AGRC schema (Township, Section, Quarter Section, Quarter-Quarter Section, Meandered Water, Special Surveys, Points.
```
Comment (steveoh) [inline _posts/2016-12-15-utah-plss-parcel-fabric-update.md]: same for here

**Comment by jpechmann** [comment]
@steveoh Updated and ready.

**Pull request merged**